schedule/src/Decoders.elm

```diff
@@ -59,6 +59,8 @@ eventDecoder =
         |> required "speaker_slugs" (list string)
         |> required "video_state" string
         |> optional "video_url" (nullable string) Nothing
+        |> optional "slides_url" (nullable string) Nothing
+        |> required "slides_filename" string
```

Comment:
This should probably be an optional like "slides_url" - we won't receive the name of the slides from the server if there aren't any slides uploaded. Also, `serialize()` on the `Event` models doesn't include any `slides_filename`, which makes the decoder confused when not receiving anything (since it's required).